featuretools/tests/synthesis/test_deep_feature_synthesis.py

```diff
@@ -412,6 +414,28 @@ def test_bad_groupby_feature(es):
                              groupby_trans_primitives=['max'])
 
 
+def test_make_rolling_features(pd_es):
```

Comment:
Might be overkill, but you could create a loop here and test all the new primitives, except maybe count which has a different naming scheme. That probably already gets tested elsewhere though.

Reply:
yeah, the test below `test_make_rolling_count_off_datetime_feature` will cover count. Happy to iterate over all the rolling primitives

Reply:
ended up parameterizing the various primitives instead of iterating over them

Reply:
Even better!